include/s3select_functions.h

```diff
@@ -1258,6 +1261,87 @@ struct _fn_like : public base_function
   }
 };
 
+struct _fn_likescape : public base_function
+{
+  void transform(std::string& s, size_t found)
```

Comment:
check against AWS the default escape char should be \\
what is the case here?
it need for single transform method (with and without escape)

Reply:
what is the defined behavior of transform(...)?

Reply:
Convert sql like expression to equivalent regex.